src/conf_mode/system-login.py

```diff
@@ -331,7 +331,7 @@ def apply(login):
             if tmp: command += f" --home '{tmp}'"
             else: command += f" --home '/home/{user}'"
 
-            command += f' --groups frr,frrvty,vyattacfg,sudo,adm,dip,disk,_kea {user}'
```

Comment:
Iirc this was necessary to access the control socket from op-mode scripts.

Reply:
Right you are! I can see op-mode commands will need this. Thanks!

Reply:
Interestingly, op-mode commands still fail. Something is going on.

Here is what I have:

- User `vyos` belongs to `_kea` group:
```
vyos@v15-c28:~$ getent group _kea
_kea:x:114:vyos
vyos@v15-c28:~$ grep _kea /etc/group
vyattacfg:x:102:tacacs15,radius_priv_user,vyos,_kea
_kea:x:114:vyos
```

- ctl-socket has 775:
```
vyos@v15-c28:~$ ls -l /run/kea/dhcp4-ctrl-socket
srwxrwxr-x 1 _kea _kea 0 Dec 29 02:56 /run/kea/dhcp4-ctrl-socket
```

- And still I see:
```
vyos@v15-c28:~$ show dhcp server leases 
Traceback (most recent call last):
  File "/usr/libexec/vyos/op_mode/dhcp.py", line 394, in <module>
    res = vyos.opmode.run(sys.modules[__name__])
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/vyos/opmode.py", line 263, in run
    res = func(**args)
          ^^^^^^^^^^^^
  File "/usr/libexec/vyos/op_mode/dhcp.py", line 261, in _wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/libexec/vyos/op_mode/dhcp.py", line 294, in show_server_leases
    lease_data = _get_raw_server_leases(family=family, pool=pool, sorted=sorted, state=state, origin=origin)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/libexec/vyos/op_mode/dhcp.py", line 88, in _get_raw_server_leases
    active_config = kea_get_active_config(inet_suffix)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/vyos/kea.py", line 318, in kea_get_active_config
    config = _ctrl_socket_command(ctrl_socket, 'config-get')
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/vyos/kea.py", line 299, in _ctrl_socket_command
    sock.connect(path)
PermissionError: [Errno 13] Permission denied
```

Reply:
A clean build solved for it.